config/boot.rb

```diff
@@ -1,6 +1,4 @@
 ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", __dir__)
 
 require "bundler/setup" # Set up gems listed in the Gemfile.
-if (RUBY_VERSION <=> "3.1.0") < 0
-  require "bootsnap/setup" # Speed up boot time by caching expensive operations.
-end
+require "bootsnap/setup" # Speed up boot time by caching expensive operations.
```

Comment:
[Added this change by otegami]
This setting is Rails default because [`depend_on_bootsnap?(!options[:skip_bootsnap])`](https://github.com/rails/rails/blob/7-0-stable/railties/lib/rails/generators/app_base.rb#L418) is usually true in default setting.

```console
$ bin/rails new --help | grep -n1 bootsnap
45-                                                           # Default: false
46:            [--skip-bootsnap]                              # Skip bootsnap gem
47-                                                           # Default: false
```
https://github.com/rails/rails/blob/d0523fc2c99ffc9899e012fb4c753a7e02b5b974/railties/lib/rails/generators/rails/app/templates/config/boot.rb.tt#L4-L6